src/explore-education-statistics-common/src/services/dataBlockService.ts

```diff
@@ -124,30 +125,30 @@ interface ResponseMetaData {
 export interface DataBlockMetadata {
   indicators: ObjectMap<LabelValueUnitMetadata>;
   filters: ObjectMap<LabelValueMetadata>;
-  timePeriods: ObjectMap<LabelValueMetadata>;
+  timePeriods: ObjectMap<TimePeriod>;
```

Comment:
We already have a util type for this called `Dictionary`

Reply:
It's not quite the same though. I'm using "name" as the key. And it's a value coming from the API

Reply:
They are the same type technically though, regardless of if they index name is called 'name' or 'key'. Would prefer we use the `Dictionary` type to be consistent here.

Reply:
Done